operations/host_spawn.go

```diff
@@ -514,6 +514,7 @@ func hostSSH() cli.Command {
 		),
 		Before: mergeBeforeFuncs(setPlainLogger, requireHostFlag),
 		Action: func(c *cli.Context) error {
+			grip.Info("Note: User must be on the VPN to gain access to the host.")
```

Comment:
Nit: since this won't be wrapped with any other text, I would use a complete sentence here.